sdks/python/apache_beam/runners/worker/statesampler_fast.pyx

```diff
@@ -190,60 +158,28 @@ cdef class StateSampler(object):
     if self.started and not self.finished:
```

Comment:
<!--thread_id:cc_162190252_t; commit:4c27aa0fbbbc31e24e3b8d6bc04128b44d18b9ec; resolved:1-->
<!--section:context-quote-->
> **robertwb** wrote:
> Is this method also common to both?

<!--section:body-->
You're right. Done.